lib/site_setting_extension.rb

```diff
@@ -186,7 +186,7 @@ def setting_metadata_hash(setting)
   end
 
   def themeable_site_settings
-    themeable.select { |_, value| value }.keys
+    themeable.select { |_, value| value }.keys.sort
```

Comment:
If you want to filter `nil` values you can use:

```rb
themeable.compact.keys.sort
```

If you want to filter blank values you can use:

```rb
themeable.compact_blank.keys.sort
```

Reply:
It's more that this is a k/v store of { key: true/false, key: true/false } so the compact won't work here